src/components/Navbar.astro

```diff
@@ -87,13 +87,13 @@ const currentPath = Astro.url.pathname;
     transition: background-color 0.3s ease;
 
     &:hover {
-      background-color: rgba($background-tertiary, 0.5);
+      background-color: rgba($background-primary, 0.5);
```

Comment:
It would be better if you add background-tertiary back instead of changing this color. The hover effect is almost invisible with this color.

Reply:
The best thing would be to make a slightly lighter version of background-secondary as background-tertiary

Reply:
Actually, it doesn't matter. It will be handled in #41